tests/test_aea.py

```diff
@@ -330,10 +328,10 @@ def test_initialize_aea_programmatically(self):
         expected_dummy_task = async_result.get(2.0)
         assert expected_dummy_task.nb_execute_called > 0
 
-        dummy_handler = self.aea.resources.handler_registry.fetch_by_protocol_and_skill(
+        dummy_handler = self.aea.resources.get_handler(
             DefaultMessage.protocol_id, dummy_skill_id
         )
-        dummy_handler_alt = self.aea.resources.handler_registry.fetch(
+        dummy_handler_alt = self.aea.resources._handler_registry.fetch(
```

Comment:
fetching handlers by name is only done in tests, I don't think we should be making this a recommended behaviour, hence no public api